pkg/reduce/reduce.go

```diff
@@ -35,10 +34,13 @@ import (
 	"github.com/numaproj/numaflow/pkg/udf/applier"
 	"github.com/numaproj/numaflow/pkg/watermark/fetch"
 	"github.com/numaproj/numaflow/pkg/watermark/publish"
+	"github.com/numaproj/numaflow/pkg/window"
 )
 
 // DataForward reads data from isb and forwards them to readloop
 type DataForward struct {
+	vertexName          string
```

Comment:
Are these used anywhere?

Reply:
it will be required when they add reduce metrics. But I think we need the replica number too. Perhaps make more sense to keep the reference of `*dfv1.Vertex`.

Reply:
sure will do.